src/libcrun/linux.c

```diff
@@ -1378,6 +1378,13 @@ create_missing_devs (libcrun_container_t *container, bool binds, libcrun_error_t
         }
     }
 
+  if (container->container_def->process && container->container_def->process->terminal)
```

Comment:
Shouldn't this be part of `needed_devs` anyways https://github.com/containers/crun/blob/main/src/libcrun/linux.c#L1135
but there is a check for `process->terminal`.

Reply:
nvm i think we need `terminal` to be present on spec.